examples/fastapi/middlewares/authed.py

```
@@ -0,0 +1,64 @@
+from ...config import TEST_DEFAULT_PLATFORM_ID, TEST_DEFAULT_USER_ID
+
+
+class AuthedMiddleware(ContextMiddleware):
+    def __init__(
+        self, app, plugins=None, default_error_response=None, *args, **kwargs
+    ):
+        super().__init__(
+            app=app,
+            plugins=plugins,
+            default_error_response=default_error_response,
+            *args,
```

Ruff (0.8.2)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)